src/prted/pmix/pmix_server_queries.c

```diff
@@ -92,8 +92,8 @@ static void _query(int sd, short args, void *cbdata)
     prte_proc_t *proct;
     pmix_proc_t *proc;
     size_t sz;
-    bool found;
-    PRTE_HIDE_UNUSED_PARAMS(sd, args);
+    bool found = false;
+    PRTE_HIDE_UNUSED_PARAMS(sd, args, found);
```

Comment:
Hmmm...so what is actually going on here is that "found" is only used in a clause protected by 
```
#ifdef PMIX_QUERY_RESOLVE_PEERS
```
and you clearly are building against an older version of PMIx. I would simply move "found" to be declared within that clause (e.g., line 848)